src/domain/entities/Tab.ts

```diff
@@ -0,0 +1,33 @@
+export interface TabParams {
```

Comment:
redundant type, use a type from codex-ui

Reply:
should we create this dependency from codex-ui `package?` entity

Reply:
it should be in a presentation level. It already depends on codex-ui Tab component

Reply:
since we are only emitting close event in tab component there is no need to store `TabParams` type in codex-ui, so it is moved to domain entities

Reply:
not removed